national-park.sql

```diff
@@ -105,11 +111,10 @@ create table Tour;
 	route		varchar(20),
 	managed_by	id_type,
 	primary key (actv_id),
-	foreign key (guide) references Employee(emp_id) on delete set null,
-	foreign key (managed_by) references Tour_Guide(emp_id) on delete set null
```

Comment:
managed_by will also be an FK right?

Reply:
yeah but who's managing a tour, Tour guide only, right? I didn't see any other entity managing a tour. Please correct me if I'm wrong.